src/main/java/Duke.java

```diff
@@ -1,10 +1,123 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+import java.util.List;
+
 public class Duke {
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
+        String logo = "   ___      _      _ __   _              \n" +
+                "  /   \\    | |    | '_ \\ | |_     __ _   \n" +
+                "  | - |    | |    | .__/ | ' \\   / _` |  \n" +
+                "  |_|_|   _|_|_   |_|__  |_||_|  \\__,_|  \n" +
+                "_|\"\"\"\"\"|_|\"\"\"\"\"|_|\"\"\"\"\"|_|\"\"\"\"\"|_|\"\"\"\"\"| \n" +
+                "\"`-0-0-'\"`-0-0-'\"`-0-0-'\"`-0-0-'\"`-0-0-' ";
         System.out.println("Hello from\n" + logo);
+
+        String greeting = "    ____________________________________________________________\n" +
+                "     Hello, Alpha here... welcome to my help centre... again :/\n" +
+                "     Would you like to explain what you want?\n" +
+                "    ____________________________________________________________";
```

Comment:
Since the logo and the greeting is constant maybe they should be static instead? And since they are static then the names should adhere to the naming conventions!